docs/native-modules.md

```diff
@@ -502,3 +513,7 @@ To access our `FancyMath` constants, we can simply call `NativeModules.FancyMath
 Calls to methods are a little different due to the asynchronous nature of the JS engine. If the native method returns nothing, we can simply call the method. However, in this case `FancyMath.add()` returns a value, so in addition to the two necessary parameters we also include a callback function which will be called with the result of `FancyMath.add()`. In the example above, we can see that the callback raises an Alert dialog with the result value.
 
 For events, you'll see that we created an instance of `NativeEventEmitter` passing in our `NativeModules.FancyMath` module, and called it `FancyMathEventEmitter`. We can then use the `FancyMathEventEmitter.addListener()` and `FancyMathEventEmitter.removeListener()` methods to subscribe to our `FancyMath::AddEvent`. In this case, when `AddEvent` is fired in the native code, `eventHandler` will get called, which logs the result to the console log.
+
+### JavaScript and Windows Runtime strings
+Note that JavaScript strings are UTF8 (i.e. `std::string`) but WinRT strings (`winrt::hstring` in C++/WinRT) are UTF16 (and therefore translate ot `std::wstring`), so when inter-operating between JavaScript and WinRT APIs, you will likely need to convert between these two formats.

```

Comment:
There's a lot of parens in this sentence and makes it a bit difficult to read. 

```suggestion
Note that JavaScript strings are UTF8 (i.e. `std::string`) but WinRT strings are UTF16 (ie. `winrt::hstring` in C++/WinRT which therefore translate to `std::wstring`), so when inter-operating between JavaScript and WinRT APIs, you will likely need to convert between these two formats.

```

Reply:
Good point :) I'll simplify it:

```suggestion
Note that JavaScript strings are UTF8 (i.e. `std::string`) but WinRT strings are UTF16 (i.e. `winrt::hstring` in C++/WinRT), so when inter-operating between JavaScript and WinRT APIs, you will need to convert between these two encodings.
```